k8s_helm_charts/README.md

```diff
@@ -0,0 +1,27 @@
+Usage:
+1. Create dedicated namespace for zdmproxy.
+
+    ```kubectl create ns zdmproxy```
+
+2. Import Secure Connect Bundle into a k8s secret called zdmproxy-scb.
+
+    ```kubectl -n zdmproxy create secret generic zdmproxy-scb --from-file=/tmp/secure-connect-origin.zip --from-file=/tmp/secure-connect-target.zip```
```

Comment:
Would recommend getting the below
```
--set origin_username="$prod_astra_user" --set origin_password="$prod_astra_escaped_password" --set target_username="$prod_astra_user" --set target_password="$prod_astra_escaped_password"
```
stuff managed like the SCB.

Reply:
Should be addressed in the latest commit.